digiroad2-api-oth/src/test/scala/fi/liikennevirasto/digiroad2/IntegrationApiSpec.scala

```diff
@@ -30,64 +30,10 @@ class IntegrationApiSpec extends FunSuite with ScalatraSuite with BeforeAndAfter
   private val integrationApi = new IntegrationApi(mockMassTransitStopService, new OthSwagger)
   addServlet(integrationApi, "/*")
 
-  def getWithBasicUserAuth[A](uri: String, username: String, password: String)(f: => A): A = {
-    val credentials = username + ":" + password
-    val encodedCredentials = Base64.encodeBase64URLSafeString(credentials.getBytes)
-    val authorizationToken = "Basic " + encodedCredentials
-    get(uri, Seq.empty, Map("Authorization" -> authorizationToken))(f)
-  }
-
   test("Should require correct authentication", Tag("db")) {
     get("/mass_transit_stops") {
       status should equal(401)
     }
-    getWithBasicUserAuth("/mass_transit_stops", "nonexisting", "incorrect") {
-      status should equal(401)
-    }
-  }
-
-  test("Get assets requires municipality number") {
```

Comment:
sä et oikein saisi tehdä näin. Nää on valideja testejä. Sun pitää poistaa authentican osa tosta vain